schemas/v0.15.0/crudapiplugin.schema.json

```diff
@@ -45,13 +45,95 @@
               "PATCH",
               "DELETE"
             ]
+          },
+          "auth": {
+            "type": "object",
+            "properties": {
+              "type": {
+                "type": "string",
+                "enum": [
+                  "None",
+                  "Entra"
+                ]
+              }
+            }
+          },
```

Comment:
Looks like we've got an extra `type` property here.

This is invalid

```json
"auth": "Entra"
```

Whereas this is valid

```json
"auth": {
    "type": "Entra"
},
```

Reply:
yup, that's a bug. Fixed it. Good catch